src/firstvoices/backend/serializers/dictionary_serializers.py

```diff
@@ -0,0 +1,17 @@
+from rest_framework import serializers
+
+from firstvoices.backend.models.part_of_speech import PartOfSpeech
+
+
+class PartsOfSpeechChildrenSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = PartOfSpeech
+        fields = ["id", "title"]
+
+
+class PartsOfSpeechSerializer(serializers.ModelSerializer):
+    children = PartsOfSpeechChildrenSerializer(many=True)
+
+    class Meta:
+        model = PartOfSpeech
+        fields = ["id", "title", "children"]
```

Comment:
Optional: the more detailed serializer could subclass the less detailed one to avoid repetition. (E.g., see the SiteSummary and SiteDetail serializers)